Python/Question_18/Sudoko.md

```diff
@@ -0,0 +1,45 @@
+##Question
+
+Given a partially filled 9×9 2D array ‘grid[9][9]’, the goal is to assign digits (from 1 to 9) to the empty cells so that every row, column, and subgrid of size 3×3 contains exactly one instance of the digits from 1 to 9.
```

Comment:
This Question forms part of the BackTracking Article Series by GeeksforGeeks. It's best to remove the Question as well as the Code.